src/coreclr/interpreter/intops.h

```diff
@@ -25,4 +31,70 @@ extern const InterpOpNameCharacters g_interpOpNameCharacters;
 
 const char* InterpOpName(int op);
 
+extern OPCODE_FORMAT const g_CEEOpArgs[];
+const char* CEEOpName(OPCODE op);
+OPCODE CEEDecodeOpcode(const uint8_t **ip);
+int CEEOpcodeSize(const uint8_t *ip, const uint8_t *codeEnd);
+
+#ifdef TARGET_64BIT
+#define INTOP_MOV_P INTOP_MOV_8
+#define INTOP_LDNULL INTOP_LDC_I8_0
+#else
+#define INTOP_MOV_P INTOP_MOV_4
+#define INTOP_LDNULL INTOP_LDC_I4_0
+#endif
+
+static inline bool InterpOpIsUncondBranch(int32_t opcode)
+{
+    return opcode == INTOP_BR;
+}
+
+static inline bool InterpOpIsCondBranch(int32_t opcode)
+{
+    return opcode >= INTOP_BRFALSE_I4 && opcode <= INTOP_BLT_UN_R8;
+}
+
+// Helpers identical to ones used by JIT
+// FIXME how to consume GET_UNALIGNED_VAL defines from pal as jit ???
```

Comment:
@janvorli Any idea how to fix this ?

Reply:
Since we don't want to build the interpreter on top of the coreclr PAL (on contrary to what the JIT currently does), we cannot include the coreclr PAL headers. 
However, there are two locations where these are defined - the palclr.h that is used for Windows and this pal_endian.h that is used for Unix. I think we should move the definitions of these functions to some common place for windows and Unix. Maybe the palclr.h would be a reasonable location. Currently, everything in it is under HOST_WINDOWS #ifdef. We can put the unix unaligned access stuff there in the #else branch or something like that.

Reply:
Interpreter seems to have access to `coreclr/minipal` and `native/minipal` as well, which serve the same purpose. Maybe move shared definitions to one of those places?

Reply:
Thanks @am11 for the suggestion, yes, that sounds like a better place. Unless we already use a similar functionality in other places than the src/coreclr, I would probably put it into the coreclr/minipal for now. But I don't have a strong preference.